### PR TITLE
Textual improvements

### PR DESCRIPTION
**Pull request by mumichae**
## Describe your changes
Minor edits on v2 documenation

## Checklist before requesting a review
- [x] I have performed a self-review of my code

- Check the correct box. Does this PR contain:
  - [ ] Breaking changes
  - [ ] New functionality
  - [ ] Major changes
  - [x] Minor changes
  - [ ] Bug fixes

- [ ] Proposed changes are described in the CHANGELOG.md

- [x] CI Preview succeeds and looks good!

### CODE REVIEW

File: documentation/contribute/add_a_dataset.qmd [hunk 1]
```diff
@@ -113,8 +119,15 @@ platforms:
 ```
 :::
 
+For more parameter options, refer to @sec-parameters
```
Comment (rcannood) [inline documentation/contribute/add_a_dataset.qmd:122]: Incomplete sentence

File: documentation/contribute/add_a_method.qmd [hunk 2]
```diff
@@ -226,31 +205,32 @@ cat src/label_projection/methods/my_method_r/script.R
 
 :::
 
-### Required sections
+The required sections are explained here in more detail:
 
-#### Imports and libraries
+### 1. Imports and libraries
```
Comment (rcannood) [inline documentation/contribute/add_a_method.qmd:210]: There is already a step 1, step 2, step 3, .... Would it make sense to use a. b. c. here?

**Review by rcannood — CHANGES_REQUESTED** [review]

File: documentation/contribute/add_a_method.qmd [hunk 3]
```diff
@@ -119,47 +121,25 @@ cat src/label_projection/methods/my_method_r/config.vsh.yaml
...
+| `functionality.info.reference`               | A bibtex reference key to the paper where the method is described.                                                                                                                                                                                                                                         |
+| `functionality.info.documentation_url`       | The url to the documentation of the method.                                                                                                                                                                                                                                                                |
+| `functionality.info.reporsitory_url`         | The repository url for the method.                                                                                                                                                                                                                                                                         |
```
Comment (rcannood) [inline documentation/contribute/add_a_method.qmd:132]: Just spotted a typo

```suggestion
| `functionality.info.repository_url`         | The repository url for the method.                                                                                                                                                                                                                                                                         |
```

File: documentation/contribute/add_a_dataset.qmd [hunk 4]
```diff
@@ -113,8 +119,15 @@ platforms:
 ```
 :::
 
+For more parameter options, refer to @sec-parameters.
```
Comment (KaiWaldrant) [inline documentation/contribute/add_a_dataset.qmd:122]: this causes the website to show `section ...` this has no meaning to the reader. I would suggest to just add a link.
```suggestion
For more parameter options, refer to [Parameters](#parameters) section.
```

File: documentation/contribute/add_a_dataset.qmd [hunk 5]
```diff
@@ -113,8 +119,15 @@ platforms:
...
+
+
 ## Step 3: Create a script
 
+Next, create a script that will generate or load the dataset.
+Here we show an example script that generates a random dataset, but check out `src/datasets/loaders` for real data examples.
+The script must ensure that that the output anndata object has the format as described in @sec-format.
```
Comment (KaiWaldrant) [inline documentation/contribute/add_a_dataset.qmd:129]: same applies for this
```suggestion
The script must ensure that that the output anndata object has the format as described in the [format](#format-of-a-raw-dataset-object) section.

```

File: documentation/contribute/add_a_dataset.qmd [hunk 6]
```diff
@@ -255,7 +268,7 @@ viash run src/datasets/loaders/myloader/config.vsh.yaml -- \
 
 <!-- TODO: write a generic dataset loader unit test and use that instead, see #103 -->
 
-## Parameters
+## Parameters {#sec-parameters}
```
Comment (KaiWaldrant) [inline documentation/contribute/add_a_dataset.qmd:271]: Not needed if changes are accepted.
```suggestion
## Parameters
```

File: documentation/contribute/add_a_dataset.qmd [hunk 7]
```diff
@@ -303,7 +316,7 @@ var <- data.frame(
 
 :::
 
-## Format of a raw dataset object
+## Format of a raw dataset object {#sec-format}
```
Comment (KaiWaldrant) [inline documentation/contribute/add_a_dataset.qmd:319]: same

```suggestion
## Format of a raw dataset object
```

File: documentation/contribute/add_a_method.qmd [hunk 8]
```diff
@@ -119,47 +121,25 @@ cat src/label_projection/methods/my_method_r/config.vsh.yaml
...
+| `functionality.info.preferred_normalization` | Which normalization method a component _prefers_. Possible values are  `r norm_methods_str` . Each value corresponds to a normalization component in the directory [ `src/datasets/normalization` ]( https://github.com/openproblems-bio/openproblems-v2/tree/main/src/datasets/normalization ).           |
+| `__merge__`                                  | The file specified in this field contains information regarding the input and output arguments of the component, as well as a unit test to ensure that the component is functioning properly. Normally you don't need to change this if you gave the right arguments to the  `create_component`  component. |
```
Comment (KaiWaldrant) [inline documentation/contribute/add_a_method.qmd:134]: Not a big fan of the full table. shouldn't we do the same for the other files ?
```suggestion
#### `functionality.name`

A unique identifier for the method. Must be written in snake case. Example: `my_new_method`.

#### `functionality.info`

Please edit this section in the config file to fill in the necessary metadata.

| Metadata field                               | Description                                                                                                                                                                                                                                                                                                |
|----------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| `pretty_name`             | A label for the method used for visualisations and documentation. Example:  `"My new method"` .                                                                                                                                                                                                            |
| `summary`                 | A one sentence summary of the method. Used for creating short overviews of the components in a task.                                                                                                                                                                                                       |
| `description`             | An explanation for how the method works. Used for creating reference documentation of a task.                                                                                                                                                                                                              |
| `reference`               | A bibtex reference key to the paper where the method is described.                                                                                                                                                                                                                                         |
| `documentation_url`       | The url to the documentation of the method.                                                                                                                                                                                                                                                                |
| `repository_url`         | The repository url for the method.                                                                                                                                                                                                                                                                         |
| `preferred_normalization` | Which normalization method a component _prefers_. Possible values are  `r norm_methods_str` . Each value corresponds to a normalization component in the directory [ `src/datasets/normalization` ]( https://github.com/openproblems-bio/openproblems-v2/tree/main/src/datasets/normalization ).           |


#### `__merge__`

The file specified in this field contains information regarding the input and output arguments of the component, as well as a unit test to ensure that the component is functioning properly. Normally you don't need to change this if you gave the right arguments to the `create_component` component.
```

File: documentation/contribute/add_a_method.qmd [hunk 9]
```diff
@@ -119,47 +121,25 @@ cat src/label_projection/methods/my_method_r/config.vsh.yaml
...
 
 ## Step 3: Add dependencies
 
 Each component has it's own set of dependencies, because different components might have conflicting dependencies.
 
-In the platforms section of the config file update the `setup` definition that describes the packages that need to be installed in the Docker image and are required for your method to run. Note that both `anndata~=0.8` and `pyyaml` are necessary Python package dependencies.
+Update the `setup` definition in the `platforms` section of the config file.
+This section describes the packages that need to be installed in the Docker image and are required for your method to run.
+Note that both `anndata~=0.8` and `pyyaml` are necessary Python package dependencies.
```
Comment (KaiWaldrant) [inline documentation/contribute/add_a_method.qmd:142]:
```suggestion
Note that both `anndata~=0.8.0` and `pyyaml` are necessary Python package dependencies.
```

**Review by KaiWaldrant — CHANGES_REQUESTED** [review]

**Comment by rcannood** [comment]
I much prefer to look at the table when the page is wide enough, but on mobile the table looks quite bad. I'll revert the tabular changes for now.

**Review by rcannood — APPROVED** [review]

**Review by KaiWaldrant — APPROVED** [review]

**Pull request merged**